app/views/layouts/_nav.html.erb

```diff
@@ -33,6 +33,13 @@
       </div>
     </nav>
 
+    <nav class="navbar-nav">
+      <h2 class="visually-hidden">Feedback link</h2>
+      <a class="nav-link nav-link-icon" href="mailto:dovetail-feedback@prx.org">
+        <span class="material-icons align-middle">feedback</span>
```

Comment:
For some reason, the vertical alignment of this looks funny to me?  With the `!` of this icon not quite aligned with the `?` next to it.  I guess that's just how this icon works.

I wonder if this needs a tooltip - how will users know what it means?  Will they just infer it from the email address?

Reply:
Added title attributes to links. Can't do much about the icon.